test/integration-test-v2.py

```diff
@@ -309,6 +313,41 @@ def test_sct_embedding():
         if sct.entry_type != x509.certificate_transparency.LogEntryType.PRE_CERTIFICATE:
             raise Exception("SCT contains wrong entry type")
 
+def test_only_return_existing_reg():
+    key = jose.JWKRSA(key=rsa.generate_private_key(65537, 2048, default_backend()))
+    net = acme_client.ClientNetwork(key, user_agent="Boulder integration tester")
+    directory = Directory.from_json(net.get(chisel2.DIRECTORY).json())
+    client = acme_client.ClientV2(directory, net)
```

Comment:
There's some duplication throughout this test of:
1) creating a `net` with a key
2) creating a `directory` with the net
3) creating a client with the directory and net

Can you hoist that into a little function?